support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -327,10 +348,6 @@ function getAmountsTestParticipations(
 	countryGroupId: CountryGroupId,
 	settings: Settings,
 ): Participations | null | undefined {
-	if (!settings.amounts) {
```

Comment:
This conditional was flagged as needless as the definition for `Settings` (`window.guardian.settings`) always has `window.guardian.settings.amounts` is not optional.